### ISSUE TITLE
Focus cycle

### ISSUE THREAD

**Issue by masakudamatsu** [masakudamatsu/masakudamatsu.dev#15]
Currently, pressing Tab key sometimes fails to show any element being focused. This can confuse the keyboard users.
